sqlglot/expressions.py

```diff
@@ -809,10 +811,10 @@ def __ror__(self, other: ExpOrStr) -> Or:
         return self._binop(Or, other, reverse=True)
 
     def __neg__(self) -> Neg:
-        return Neg(this=_wrap(self, Binary))
+        return Neg(this=_wrap(self.copy(), Binary))
```

Comment:
Can also make it so that `_wrap` copies `expression` by default whenever it creates a `Paren` that wraps it.